cluster/gce/config-default.sh

```diff
@@ -36,6 +36,11 @@ NODE_IMAGE_PROJECT=${KUBE_GCE_NODE_PROJECT:-"${MASTER_IMAGE_PROJECT}"}
 CONTAINER_RUNTIME=${KUBE_CONTAINER_RUNTIME:-docker}
 RKT_VERSION=${KUBE_RKT_VERSION:-0.5.5}
 
+# If the distro is CoreOS, kubelet will run as a systemd service.
+if [[ "${OS_DISTRIBUTION}" == "coreos" ]]; then
```

Comment:
oops, sorry,missed exactly where you're checking this from - I doubt you want to expose this from within the kubelet.